.github/workflows/test-generated-project.yml

```
@@ -39,17 +39,12 @@ jobs:
       - name: Generate project
         run: . ./scripts/generateAdvancedAWS.sh
 
-      - name: Install Terraform
-        uses: hashicorp/setup-terraform@v2
-        with:
-          terraform_version: ${{ env.TERRAFORM_VERSION }}
+      - name: Install dependencies in .tool-versions
```

```suggestion
      - name: Install dependencies from .tool-versions
```

(we don't install the dependencies in the `.tool-versions` file, but we use the file to install the dependencies).